test/gateway/safe_core/commit_state_root.js

```diff
@@ -29,7 +29,7 @@ const zeroBytes =
   "0x0000000000000000000000000000000000000000000000000000000000000000";
 
 contract('SafeCore.commitStateRoot()', function (accounts) {
-
+  
```

Comment:
Please no extra spaces at line endings. You should be able to set-up your IDE to delete those automatically when saving.

Reply:
I corrected the IDE settings.